packages/force_update/README.md

```diff
@@ -3,12 +3,9 @@
 This is an internal Leancode package for quickly implementing Force Update. To use it, just wrap your `MaterialApp` in a `ForceUpdateGuard`
 widget and pass an instance of CQRS.
 
-# Behavior
+# Behaviour
```

Comment:
Use American spelling

Reply:
why? do we have such convention? I changed this for unification

Reply:
I think that we usually use American spelling, from the projects I've seen